src/app/_components/Links/SocialLinksList.js

```diff
@@ -1,7 +1,18 @@
 import Link from 'next/link';
 import PropTypes from 'prop-types';
+import { getLinksList } from '@components/Links/linksActions';
+import { NavigationUrl } from '@prisma/client';
+import { Facebook, Instagram } from '@icons';
+
+export async function SocialLinksList({ className }) {
+  const { socialMediaList } = await getLinksList();
```

Comment:
If you move this call to the layout, then we can remove it here

Reply:
Have done it!